client/ngapp/views/country-view.html

```diff
@@ -7,15 +7,39 @@
       </h1>
     </div>
 
-    <p>
-      Here's what we know about anomalies with internet connections located in <span class="capitalize">{{ countryName | lowercase }}</span>.
-    </p>
+    <ul class="toc">
```

Comment:
I implementing the scrolling behaviour by having a custom directive. I also tried a series of techniques using `$anchorScroll` and variations on that, but they all presented the problems you described.

You can see this fix this commit: https://github.com/TheTorProject/ooni-explorer/commit/7c78c4f7efb62a72cf80471367045d0b64ec418f